libV2/tasks/DiskUsage.js

```diff
@@ -84,16 +86,19 @@ class MonitorDiskUsage extends BaseTask {
             return;
         }
 
-        moduleLogger.info('deleting metrics', {
-            start: oldestTimestamp, stop: endTimestamp,
-        });
-
-        await this.withWarp10(async warp10 =>
-            warp10.delete({
-                className: '~.*',
-                start: oldestTimestamp - 1,
-                end: endTimestamp,
-            }));
+        await async.eachSeries(
+            sliceTimeRange(oldestTimestamp - 1, endTimestamp, expirationChunkDuration),
+            async ([start, end]) => {
+                moduleLogger.info('deleting metrics',
```

Comment:
It could be better to log `start` and `end` instead, or in addition to oldest/end timestamps

Reply:
ah yeah thats what I meant to do

Reply:
don't forget to change it :wink: